packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -2961,7 +2961,11 @@ export class ObjectsInRedisClient {
     ): Promise<ioBroker.CallbackReturnTypeOf<ioBroker.GetObjectCallback>>;
     // no options but cb
     getObject(id: string, callback: ioBroker.ErrorCallback): void;
-    getObject(id: string, options?: any, callback?: ioBroker.ErrorCallback) {
+    getObject(
+        id: string,
+        options?: any,
+        callback?: ioBroker.ErrorCallback
+    ): void | Promise<ioBroker.CallbackReturnTypeOf<ioBroker.GetObjectCallback>> {
```

Comment:
By defining all overloads this way, you're missing out on the ID to object type inference I've added in @types/iobroker. I suggest defining them in a similar way I did for `getForeignObject` here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/06f38fe1967d69ed164552b55674bf2d5269e7a1/types/iobroker/index.d.ts#L647-L651